managed-pointer/cpp-20/D0567R1.md

```diff
@@ -229,94 +391,105 @@ auto gpuExecutor = gpuContext.executor();
 auto gpuAllocator = gpuContext.allocator();
 
 /* Construct a managed_ptr ptraA allocated on the host CPU */
-std::experimental::execution::managed_ptr<float> ptrA(1024);
+execution::managed_ptr<my_data> ptrA();
```

Comment:
The empty parenthesis are not needed.
Does this calls the constructor? 
If so, is the constructor called on each allocation on different memory spaces or only once and then replicated?
Depending on this, the managed_ptr may be restricted to trivial constructable types or not, but it is not specified